src/main/resources/migrations/0012_alter_table_webhook_messages_add_last_delivery_status.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE webhook_messages
+ADD COLUMN last_delivery_status VARCHAR(64) check (last_delivery_status in ('SUCCESSFUL', 'FAILED', 'PENDING', 'WILL_NOT_SEND'));
+
+CREATE INDEX IF NOT EXISTS last_delivery_status_idx on webhook_messages(last_delivery_status)
```

Comment:
Should we create it `concurrently`?

Reply:
I think we'd be OK to not create it concurrently as the column will be definitionally empty (having just been created if this migration is running) but happy to add it as best practice